extensions/google-translate/src/translate.tsx

```diff
@@ -31,35 +31,33 @@ export default function Command(): ReactElement {
 
     const promises = Promise.all([
       translate(toTranslate, {
-        from: preferences.lang1,
+        from: preferences.lang1 === "auto" ? undefined : preferences.lang1,
         to: preferences.lang2,
       }),
-      translate(toTranslate, {
-        from: preferences.lang2,
-        to: preferences.lang1,
-      }),
```

Comment:
Sorry @tangerine1202 this was my mistake, I missed that it was working like that earlier so you should not remove it

Reply:
It's fine. Do you need me to fix it?

Reply:
I did it in this PR: https://github.com/raycast/extensions/pull/4667